### PR TITLE
{bio}[foss/2018b] PRINSEQ v0.20.4 w/ Perl 5.28.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/00320d107f890e4b52da9666d8d14e24 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2532.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f579a7a27b509b137f92052f8cf4b905 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2105.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2f46861f94ee5b36ac92625234408dc1 for a full test report.

File: easybuild/easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-foss-2018b-Perl-5.28.0.eb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        'source_tmpl': 'Cairo-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+        'checksums': ['e64803018bc7cba49e73e258547f5378cc4249797beafec524852140f49c45c4'],
+    }),
+    ('Statistics::PCA', '0.0.1', {
+        'source_tmpl': 'Statistics-PCA-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DS/DSTH/'],
+        'checksums': ['f8adb10b00232123d103a5b49161ad46370f47fe0f752e5462a4dc15f9d46bc4'],
+    }),
+    ('MIME::Base64', '3.15', {
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-foss-2018b-Perl-5.28.0.eb]: shouldn't some of these be in the Perl easyconfig directly? They are general purpose?

Comment (boegel) [reply easybuild/easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-foss-2018b-Perl-5.28.0.eb]: The need never arose, beyond here?

I don't know of anything else that requires this one...

**Pull request merged**